servers/rendering/shader_language.cpp

```diff
@@ -9733,7 +9733,13 @@ Error ShaderLanguage::_parse_shader(const HashMap<StringName, FunctionInfo> &p_f
 
 					if (tk.type == TK_COMMA) {
 						tk = _get_token();
-						//do none and go on
+#ifdef DISABLE_DEPRECATED
+						// Disallow trailing comma.
+						if (tk.type == TK_PARENTHESIS_CLOSE) {
+							_set_error(RTR("Expected a valid data type for argument."));
```

Comment:
Picked this error as it matches what happens if you do `void foo(,)`

Reply:
Would a more user-friendly error be better, though? E.g. "(Remove the trailing comma?)"

Reply:
Could add 🙂 considering a warning as well so working on the wording, thank you!